app/services/github_service.rb

```diff
@@ -29,13 +29,19 @@ def handle_event
   end
 
   def resolve_event_name(event)
+    return event.gsub('issue_', 'pull_request_') if pull_request_comment?
+
     return event.gsub('pull_request_', '') unless event == 'pull_request'
 
     event
   end
 
   def handleable_event?
-    Event::TYPES.include?(@event)
+    Event::TYPES.include?(@event) || pull_request_comment?
```

Comment:
Is there any reason to not have `pull_request_comment` included in `Event::TYPES` ?

Reply:
Actually, I decided to go with this implementation because the event itself has the name "issue_comment", and it is used not only for PR comments but for Github Issues comments. And we don't want to include that ones but the ones that have "pull_request" in their json.